blockchain/README.md

```diff
@@ -26,35 +26,51 @@ As described before: The Trubudget Blockchain is created one by a master, which
+| EMAIL_PORT                 | no       |                  | The port address from the email-notification service.                                                                                                            |
+| EMAIL_SSL                  | no       | false            | If set to `true` the connection between blockchain and email-notification service is https instead of http                                                       |
+| NOTIFICATION_PATH          | no       | ./notifications/ | The path where notification files shall be saved on the blockchain environment                                                                                   |
+| NOTIFICATION_MAX_LIFETIME  | no       | 24               | This number configure how long notifications shall be saved in the NOTIFICATION_PATH in hours                                                                    |
+| NOTIFICATION_SEND_INTERVAL | no       | 10               | This number configure in which interval the notifications in the NOTIFICATION_PATH should be checked and send                                                    |
+| JWT_SECRET                 | no       |                  | The `JWT_SECRET` is only required if the Email feature is enabled. It is used to authenticate the blockchain at the email-service, so it can send notifications. |
+
+#### Kubernetes
+
+| Env Variable | Required | Default Value | Description                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| ------------ | -------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| EXPOSE_MC    | no       | false         | If set to true Trubudget tries to expose it's multichain via kubernetes' ingress. First a kubernetes config file is searched in `/.kube/config`. If not found and the MC is deployed in a cluster it is search for the service account to get the information. After configuration the `externalip` arg is set to the ip of the service' ingress of the configured cluster SERVICE_NAME and NAMESPACE are required to access the ingress of the service. |
```

Comment:
If set to true**,** Trubudget tries to expose it's multichain via kubernetes' ingress


If not found and the MC is deployed in a cluster**, it is searching** for the service account to get the information